bin/spice/pkg/github/runtime_release.go

```diff
@@ -125,3 +137,31 @@ func has_metal_device() (bool, error) {
 	}
 	return strings.Contains(string(output), "Metal Support: Metal"), nil
 }
+
+func has_cuda_device() (bool, error) {
+	if runtime.GOOS != "linux" {
+		return false, nil
+	}
+
+	slog.Debug("On Linux, running `nvidia-smi --query-gpu=name --format=csv,noheader` to determine hardware")
+	cmd := exec.Command("nvidia-smi", "--query-gpu=name", "--format=csv,noheader")
+	if err := cmd.Start(); err != nil {
+		return false, fmt.Errorf("failed to start `nvidia-smi` command: %w", err)
```

Comment:
Will start return an error if the command isn't found?

Reply:
Fixed